corona_stats/README.md

```diff
@@ -22,5 +22,3 @@ Available commands:
 `/nc corona_stats <Country Name | Abbreviation > -r <State Name | Abbreviation>`  -- to get stats for a specific state.
 
 `/nc corona_stats <Country Name | Abbreviation > -r <District Name>`  -- to get stats for a specific district.
-
-[Read More](https://nimbella.com/blog/get-live-coronavirus-stats-in-slack-with-nimbella-commander/)
```

Comment:
I think it's a good thing to keep a link to our blog post. We can reframe it to something like "Read our blog on corona stats in Slack to learn more."

Reply:
thanks, added it. I thought it to be slack specific and wanted to keep the readme generic. Further evolution will fix things.